src/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './draw';
 export * from './featureMatching';
 export * from './filters';
 export * from './geometry';
+export * from './utils/geometry/filterPoints';
```

Comment:
We should not expose lower-level files from here. If `filterPoints` is meant to be public API (but should it really? The name is very generic for a much more specific function), it should be exported from the `geometry/index.ts` file.

Reply:
I also have an issue with the name and wasn't sure if it made sense to export it from the library vs let the algo be run with certain options of the watershed function.

If this is meant to be used to get starting points for watershed algorithm, why not either name it after that (like `findWatershedPoints` or find something in the middle in terms of genericness (like `getFilteredLocalExtrema`).


@lpatiny WDYT? I believe you discussed this with @EscapedGibbon